src/gatling/simulations/uk/gov/hmcts/reform/CRD/performance/scenarios/OrgRoleMappingScenario.scala

```diff
@@ -1,5 +1,30 @@
 package uk.gov.hmcts.reform.CRD.performance.scenarios
 
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.CRD.performance.scenarios.utils.{ASBHelper, Environment}
+import uk.gov.hmcts.reform.CRD.performance.scenarios.utils.Environment._
+
 object OrgRoleMappingScenario {
 
-}
+  val url = "sb://rd-servicebus-perftest.servicebus.windows.net"
+  val keyName = "SendAndListenSharedAccessKey"
+  //PerfTest
+  val keyVale = "1MWyKwjaplH9PyWub8zKO8cAreD+5f9gzKjjESVeYFM="
```

Comment:
what's this? should it be available in a repo?

Reply:
This is removed and should be picked up from the vault.